notebooks/book2/05/minimize_kl_divergence.ipynb

```diff
@@ -0,0 +1,230 @@
+{
```

Comment:
<p>Here you should add two plots for loss vs iterations.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>